enterprise/server/util/proxy/proxy.go

```diff
@@ -0,0 +1,19 @@
+package proxy_util
```

Comment:
rename from `enterprise/server/util/proxy/proxy.go` => `enterprise/server/util/proxy_util/proxy_util.go` to match the package name